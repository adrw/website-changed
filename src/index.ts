export const testBlazeVacancies = false

export const websites: string[] = ["https://adrw.xyz"].concat(
  testBlazeVacancies
    ? [
        "http://www.blazeproperties.ca/index.php/properties/abercrombie-place",
        "http://www.blazeproperties.ca/index.php/properties/belmont-village",
        "http://www.blazeproperties.ca/index.php/properties/camelot",
        "http://www.blazeproperties.ca/index.php/properties/cripple-creek",
        "http://www.blazeproperties.ca/index.php/properties/cripple-creek",
        "http://www.blazeproperties.ca/index.php/properties/tivoli",
        "http://www.blazeproperties.ca/index.php/properties/union-lane"
      ]
    : []
)
