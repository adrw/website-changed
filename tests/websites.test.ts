import { testBlazeVacancies, websites } from "../src"
import axios from "axios"
const htmlParser = require("html-parser")

if (testBlazeVacancies) {
  describe("blaze-vacancies", () => {
    it("blaze-vacancies", () => {
      return axios
        .get("http://www.blazeproperties.ca/index.php/vacancies")
        .then(resp => {
          expect(resp.data).toBeDefined()
          let vacanciesH2 = false
          let vacanciesList = false
          let vacanciesSnapshot: string[] = []
          htmlParser.parse(resp.data, {
            openElement: (name: string) => {
              if (name == "h2") {
                vacanciesH2 = true
              }
            },
            closeElement: (name: string) => {
              if (vacanciesH2 && name === "h2") {
                vacanciesList = true
              }
            },
            text: (value: string) => {
              if (vacanciesH2 && value !== "Vacancies") {
                vacanciesH2 = false
              }
              if (vacanciesList && value === "Sorry, no vacancy.") {
                vacanciesList = false
              }
              if (
                vacanciesList &&
                value.trim().length !== 0 &&
                value.trim() !== ","
              ) {
                vacanciesSnapshot.push(value)
              }
            }
          })
          expect(vacanciesSnapshot).toMatchSnapshot()
        })
    })
  })
}

websites.map(url => {
  describe("websites", () => {
    it(url, () => {
      return axios.get(url).then(resp => {
        expect(resp.data).toBeDefined()
        expect(resp.data).toMatchSnapshot()
      })
    })
  })
})
