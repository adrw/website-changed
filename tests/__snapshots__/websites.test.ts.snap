// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`websites https://adrw.xyz 1`] = `

<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible"
          content="ie=edge"
    >
    <meta name="viewport"
          content="width=device-width, initial-scale=1, shrink-to-fit=no"
    >
    <style data-href="/0.9cfcfc49081c4637ddf6.css">
      /*! modern-normalize | MIT License | https://github.com/sindresorhus/modern-normalize */html{box-sizing:border-box}*,:after,:before{box-sizing:inherit}:root{-moz-tab-size:4;-o-tab-size:4;tab-size:4}html{-webkit-text-size-adjust:100%;line-height:1.15}body{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol;margin:0}hr{height:0}abbr[title]{-webkit-text-decoration:underline dotted;text-decoration:underline dotted}b,strong{font-weight:bolder}code,kbd,pre,samp{font-family:SFMono-Regular,Consolas,Liberation Mono,Menlo,Courier,monospace;font-size:1em}small{font-size:80%}sub,sup{font-size:75%;line-height:0;position:relative;vertical-align:baseline}sub{bottom:-.25em}sup{top:-.5em}button,input,optgroup,select,textarea{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,select{text-transform:none}[type=button],[type=reset],[type=submit],button{-webkit-appearance:button}[type=button]::-moz-focus-inner,[type=reset]::-moz-focus-inner,[type=submit]::-moz-focus-inner,button::-moz-focus-inner{border-style:none;padding:0}[type=button]:-moz-focusring,[type=reset]:-moz-focusring,[type=submit]:-moz-focusring,button:-moz-focusring{outline:1px dotted ButtonText}fieldset{padding:.35em .75em .625em}legend{padding:0}progress{vertical-align:baseline}[type=number]::-webkit-inner-spin-button,[type=number]::-webkit-outer-spin-button{height:auto}[type=search]{-webkit-appearance:textfield;outline-offset:-2px}[type=search]::-webkit-search-decoration{-webkit-appearance:none}::-webkit-file-upload-button{-webkit-appearance:button;font:inherit}summary{display:list-item}
    </style>
    <meta name="generator"
          content="Gatsby 2.0.35"
    >
    <script>
      window.__EMOTION_CRITICAL_CSS_IDS__ = ["pn26jt","zf0iqh","1khnq5v","8gr2bf","44457s","b95f0i","pcbh56","fkj7ih","1pbhmre"];
    </script>
    <style>
      html{box-sizing:border-box;}*,*::before,*::after{box-sizing:inherit;}html{font-size:16px !important;line-height:1.45 !important;}body{width:100%;overflow-x:hidden;overflow-y:scroll;font-family:Avenir Next,-apple-system,BlinkMacSystemFont,"Segoe UI","Roboto","Oxygen","Ubuntu","Cantarell","Fira Sans","Droid Sans","Helvetica Neue",Arial,sans-serif;color:#000;background-color:#fff;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%;background-image:url("img/00015c.png");}a{color:#333;-webkit-text-decoration:none;text-decoration:none;border-bottom:2px solid;}a:hover,a:focus{-webkit-text-decoration:none;text-decoration:none;border-bottom:4px solid;}img{max-width:100%;object-fit:contain;position:relative;}figure{margin:2rem 0;}figcaption{font-size:80%;}table{width:100%;margin-bottom:1rem;border:1px solid #f5f3f7;font-size:85%;border-collapse:collapse;}td,th{padding:.25rem .5rem;border:1px solid #f5f3f7;}th{text-align:left;}tbody tr:nth-child(odd) td{background-color:#fbfafc;}tbody tr:nth-child(odd) tr{background-color:#fbfafc;}h1,h2,h3,h4,h5,h6{margin-top:1.414rem;margin-bottom:.5rem;color:#000;font-weight:500;line-height:1.2;text-rendering:optimizeLegibility;}h1{margin-top:0;font-size:2.441rem;}h2{font-size:1.953rem;}h3{font-size:1.563rem;}h4,h5,h6{font-size:1.25rem;}p{margin-top:0;margin-bottom:1rem;}strong{color:#000;}ul,ol,dl{margin-top:0;margin-bottom:1rem;}dt{font-weight:bold;}dd{margin-bottom:.5rem;}hr{position:relative;margin:1.5rem 0;border:0;border-top:1px solid #f5f3f7;}blockquote{margin:.8rem 0;padding:.5rem 1rem;border-left:.25rem solid #f5f3f7;color:rgba(0,0,0,0.54);}blockquote p:last-child{margin-bottom:0;}@media (min-width:48em){blockquote{padding-right:5rem;padding-left:1.25rem;}}.css-zf0iqh{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;min-height:100vh;}.css-1khnq5v{height:60px;padding:0 1.5rem;color:rgba(51,51,51,0.5);}.css-8gr2bf{position:relative;margin-left:auto;margin-right:auto;width:auto;max-width:60em;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:row;-ms-flex-direction:row;flex-direction:row;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;height:100%;}.css-44457s{color:#333;font-size:1.5rem;font-weight:600;}.css-44457s:hover,.css-44457s:focus{-webkit-text-decoration:none;text-decoration:none;}.css-b95f0i{display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;-webkit-flex:1;-ms-flex:1;flex:1;}.css-pcbh56{display:block;-webkit-flex:1;-ms-flex:1;flex:1;position:relative;padding:1.5rem;margin-bottom:3rem;}.css-fkj7ih{position:relative;margin-left:auto;margin-right:auto;width:auto;max-width:60em;}.css-1pbhmre{margin:10% 0;}
    </style>
    <link rel="canonical"
          href="https://www.adrw.xyz//"
          data-baseprotocol="https:"
          data-basehost="www.adrw.xyz"
    >
    <title data-react-helmet="true">
      ADRW Digital | Andrew Paradi Alexander
    </title>
    <meta data-react-helmet="true"
          name="description"
          content="ADRW"
    >
    <meta data-react-helmet="true"
          name="keywords"
          content="gatsbyjs, gatsby, javascript, sample, something"
    >
    <link as="script"
          rel="preload"
          href="/0-c55121b1eaced0421d00.js"
    >
    <link as="script"
          rel="preload"
          href="/component---src-pages-index-tsx-0bb2ae0b096ab9724589.js"
    >
    <link as="script"
          rel="preload"
          href="/app-e89c1ef3ffeab20ee047.js"
    >
    <link as="script"
          rel="preload"
          href="/webpack-runtime-959b834dd9212f461642.js"
    >
    <link rel="preload"
          href="/static/d/173/path---index-6a9-NZuapzHg3X9TaN1iIixfv1W23E.json"
          as="fetch"
          crossorigin="use-credentials"
    >
  </head>
  <body>
    <div id="___gatsby">
      <div style="outline:none"
           tabindex="-1"
           role="group"
      >
        <div class="css-zf0iqh e17su9850">
          <header class="css-1khnq5v e14yya1q0">
            <div class="e14yya1q1 css-8gr2bf e1c6zxln0">
              <a aria-current="page"
                 class
                 href="/"
              >
                <img>
              </a>
              <a aria-current="page"
                 class="css-44457s e14yya1q2"
                 href="/"
              >
                ADRW Digital | Andrew Paradi Alexander
              </a>
            </div>
          </header>
          <main class="css-b95f0i e1qy7fsr0">
            <div class="css-pcbh56 e20rco30">
              <div class="css-fkj7ih e1c6zxln0">
                <h1>
                  Andrew Paradi Alexander
                </h1>
                <p>
                  I&#x27;m a software engineer at Square building
                  <a href="https://github.com/square/misk">
                    Misk
                  </a>
                  and
                  <a href="https://github.com/square/misk-web">
                    Misk-Web
                  </a>
                  scalable platforms for the Cash App. I studied Computer Science at University of Waterloo.
                </p>
                <h5>
                  <a href="http://adrw.ch/about">
                    About
                  </a>
                </h5>
                <h5>
                  <a href="http://adrw.ch/experience">
                    Experience
                  </a>
                </h5>
                <h5>
                  <a href="http://adrw.ch/projects">
                    Projects
                  </a>
                </h5>
                <h5>
                  <a href="http://adrw.ch/blog">
                    Blog
                  </a>
                </h5>
                <h5>
                  <a href="http://adrw.ch/contact">
                    Contact
                  </a>
                </h5>
                <h5>
                  <a href="http://resume.adrw.ch">
                    Resume
                  </a>
                </h5>
                <h5>
                  <a href="mailto:andrew@adrw.ch">
                    andrew@adrw.xyz
                  </a>
                </h5>
                <h5>
                  <a href="http://g.adrw.ch">
                    GitHub @adrw
                  </a>
                </h5>
                <h5>
                  <a href="http://li.adrw.ch/">
                    LinkedIn
                  </a>
                </h5>
                <br>
                <br>
                <br>
                <h1>
                  ADRW Digital
                </h1>
                <p>
                  I have experience helping startups successfully describe their product and team to customers, VCs, and judging panels. If you&#x27;re looking to improve your hit rate in your next product launch, funding round, or competition, let&#x27;s talk.
                </p>
                <p>
                  Check out past projects below. If you&#x27;re looking to improve your hit rate in your next product launch, funding round, or competition,
                  <a href="mailto:contact@adrw.xyz">
                    let&#x27;s talk
                  </a>
                  .
                </p>
                <ul>
                  <li>
                    <a href="http://www.adrw.ch/blog/videostream-how-growth-starts-with-great-customer-support">
                      Videostream
                    </a>
                    : Optimized new website &amp; sales copy to boost weekly downloads 32% + active users 12% for 1.2M user app
                  </li>
                  <li>
                    <a href="http://www.adrw.ch/project/atomic">
                      Atomic.VC
                    </a>
                    : Built in 16 weeks from scratch 7 robust apps (5 React/Redux/KoaJS, 2 iOS) for new VC vetted &amp; funded ideas
                  </li>
                  <li>
                    <a href="http://www.adrw.ch/project/teknically-webplio">
                      Teknically
                    </a>
                    : Developed presentations and written descriptions of Webplio product to win over $65,000 in different competitions and grants.
                  </li>
                </ul>
                <h5>
                  <a href="http://www.adrw.ch/projects/">
                    Other Projects
                  </a>
                </h5>
                <h5>
                  <a href="https://github.com/adrw">
                    GitHub Projects
                  </a>
                </h5>
              </div>
            </div>
            <div class="css-fkj7ih e1c6zxln0">
              <div class="css-pcbh56 e20rco30">
                <div class="css-1pbhmre eaf1f5d0">
                  ©
                  <a aria-current="page"
                     class
                     href="/"
                  >
                    ADRW Digital
                  </a>
                  2009-
                  2019
                </div>
              </div>
            </div>
          </main>
        </div>
      </div>
    </div>
    <script id="gatsby-script-loader">
      /*
      <![CDATA[*/window.page={"componentChunkName":"component---src-pages-index-tsx","jsonName":"index","path":"/"};window.dataPath="173/path---index-6a9-NZuapzHg3X9TaN1iIixfv1W23E";/*]]>*/
    </script>
    <script id="gatsby-chunk-mapping">
      /*
      <![CDATA[*/window.___chunkMapping={"app":["/app-e89c1ef3ffeab20ee047.js"],"component---src-templates-page-tsx":["/component---src-templates-page-tsx-a92d4205bd43c8df77b7.js"],"component---src-pages-404-tsx":["/component---src-pages-404-tsx-4013f6dc8decf489ea55.js"],"component---src-pages-index-tsx":["/component---src-pages-index-tsx-0bb2ae0b096ab9724589.js"],"component---src-pages-page-2-tsx":["/component---src-pages-page-2-tsx-01d968ff982266d7107d.js"]};/*]]>*/
    </script>
    <script src="/webpack-runtime-959b834dd9212f461642.js"
            async
    >
    </script>
    <script src="/app-e89c1ef3ffeab20ee047.js"
            async
    >
    </script>
    <script src="/component---src-pages-index-tsx-0bb2ae0b096ab9724589.js"
            async
    >
    </script>
    <script src="/0-c55121b1eaced0421d00.js"
            async
    >
    </script>
  </body>
</html>

`;
